src/constant/constant.ts

```diff
@@ -17,3 +17,5 @@ export enum UserStatus {
   Inactive = "inactive",
   Invited = "invited",
 }
+
+export const ATTENDANCE_PATH = "/attendance";
```

Comment:
We don't need that. Route should be defined in routes enum.

Reply:
Okay